doc/example/test_mtssl.sh

```diff
@@ -10,6 +10,8 @@ if [ "`which convolve-mtss-rotamers.py`" = "" ]; then
     exit 2
 fi
 
+useNOelectron=1
```

Comment:
Is there a reason for representing a boolean value with 1 instead of `True`? This should be either `True` or `False`.